docs/guide/quickstart.qmd

```diff
@@ -0,0 +1,763 @@
+- **Evaluate the Impact of Demands**: Introduce water demand (such as irrigation) and assess their effects on the river basin.
+- **Modify and Update Models**: Learn how to update existing models with new data and changes.
+- **Analyze Simulation Results**: Use built-in tools to analyze and interpret the results of your simulations.
+
+# Starting RIBASIM
+## System requirements
+Before installing Ribasim, ensure your system meets the following requirements:
+
+- Operating System: Windows 10 or later, or Linux (latest distributions)
+- Processor: x86-64 (64-bit)
+- RAM: 4 GB minimum, 8 GB recommended
+- Hard Drive: 1GB of free space
+
+## Installation
+1. Download Ribasim: Obtain the Ribasim 9 installation package from the official website: [Ribasim - Installation](deltares.github.io) under chapter '2 Download':
+
+* For Windows download: ‘ribasim_windows.zip’
+* For Linux: ‘ribasim_linux.zip’
+
+2. Unpack the .zip archive: It is important to keep the contents of the zip file organized within a single directory. The Ribasim executable can be found in the directory;
+3. Check installation: To check whether the installation was performed successfully, in cmd go to the executable path and type ribasim with no arguments in the command line. This will give the following message:
+
+```batch
+error: the following required arguments were not provided:
+ <TOML_PATH>
+Usage: ribasim <TOML_PATH>
+For more information, try '--help'.
+```
+
+4. We use a command line interface (CLI) to install our Ribasim packages. To install Ribasim open PowerShell or Windows command prompt and write:
+
+```batch
+conda install Ribasim
+```
+or
+
+```batch
+mamba install Ribasim
+```
+## Data preparation
+Download the Crystal_Basin.zip file from the website. Extract Crystal_Basin.zip and place it in the same directory as your Ribasim installation. This folder includes:
+
+- QuickStartGuide.pdf
+- data: Contains data inputs such as time series needed for running the case.
+Additionally, your Python model (.py) and eventually the output files will also be saved in this folder.
+
+# Modual 1 - Crystal River Basin
+We will examine a straightforward example of the Crystal River Basin, which includes a main river and a single tributary flowing into the sea (see @fig-crystal-basin).
+An average discharge of 44.45 m3/s is measured at the confluence.
+In this module, the basin is free of any activities, allowing the model to simulate the natural flow.
+The next step is to include a demand (irrigation) that taps from a canal out of the main river.
+
+![Crystal Basin based on natural flow](https://github.com/user-attachments/assets/e7c4e14c-07ca-416a-877f-9ea0459bdf3d){fig-align="left" #fig-crystal-basin}
+
+After this module the user will be able to:
+
+- Build a river basin model from scratch
+- Understand the functionality of the  ‘demand’ and ‘basin’ nodes
+- Generate overview of results
+- Evaluate the simulation results
```

Comment:
It would be nice if this looked the same as the learning objectives above, but the additional explanation present above is missing here